errors/es-mx.po

```diff
@@ -0,0 +1,1474 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: squid\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-09-16 17:53+1300\n"
+"PO-Revision-Date: 2010-08-17 06:57+0200\n"
+"Last-Translator: Javier <javier@aex.mx>\n"
+"Language-Team: Spanish (Mexico) <es-mx@li.org>\n"
+"Language: es-mx\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Pootle 2.1.6\n"
+
+
+#: templates/ERR_SECURE_CONNECT_FAIL+html.body.div.div.blockquote.p:31-1
+msgid "%D"
+msgstr ""
+
+#: templates/ERR_SECURE_CONNECT_FAIL+html.body.div.div.blockquote.pre:30-1
+msgid "%E (TLS code: %x)"
+msgstr ""
+
+#: templates/ERR_FTP_FAILURE+html.body.div.blockquote.pre:30-1
+#: templates/ERR_FTP_FORBIDDEN+html.body.div.blockquote.pre:30-1
+#: templates/ERR_FTP_NOT_FOUND+html.body.div.blockquote.pre:30-1
+#: templates/ERR_FTP_PUT_ERROR+html.body.div.blockquote.pre:30-1
+#: templates/ERR_FTP_UNAVAILABLE+html.body.div.blockquote.pre:31-1
+msgid "%F"
+msgstr ""
+
+#: templates/ERR_INVALID_REQ+html.body.div.blockquote.pre:24-1
+#: templates/ERR_INVALID_RESP+html.body.div.blockquote.pre:24-1
+msgid "%R"
+msgstr ""
+
+#: templates/ERR_ESI+html.body.div.blockquote.pre:29-1
+msgid "%Z"
+msgstr ""
+
+#: templates/ERR_FTP_FAILURE+html.body.div.blockquote.pre:25-1
+#: templates/ERR_FTP_FORBIDDEN+html.body.div.blockquote.pre:25-1
+#: templates/ERR_FTP_NOT_FOUND+html.body.div.blockquote.pre:25-1
+#: templates/ERR_FTP_PUT_ERROR+html.body.div.blockquote.pre:25-1
+#: templates/ERR_FTP_UNAVAILABLE+html.body.div.blockquote.pre:26-1
+msgid "%f"
+msgstr ""
+
+#: templates/ERR_FTP_FAILURE+html.body.div.blockquote.pre:31-1
+#: templates/ERR_FTP_FORBIDDEN+html.body.div.blockquote.pre:31-1
+#: templates/ERR_FTP_NOT_FOUND+html.body.div.blockquote.pre:31-1
+#: templates/ERR_FTP_UNAVAILABLE+html.body.div.blockquote.pre:32-1
+msgid "%g"
+msgstr ""
+
+#: templates/error-details.txt+X509_V_ERR_OUT_OF_MEM.detail:73
+msgid "%ssl_error_descr"
+msgstr "%ssl_error_descr"
+
+#: templates/error-details.txt+X509_V_ERR_INVALID_CA.detail:101
+#: templates/error-details.txt+X509_V_ERR_SUBJECT_ISSUER_MISMATCH.detail:121
+#: templates/error-details.txt+X509_V_ERR_AKID_ISSUER_SERIAL_MISMATCH.detail:129
+msgid "%ssl_error_descr: %ssl_ca_name"
+msgstr "%ssl_error_descr: %ssl_ca_name"
+
+#: templates/error-details.txt+SQUID_ERR_SSL_HANDSHAKE.detail:5
+#, fuzzy
+msgid "%ssl_error_descr: %ssl_lib_error"
+msgstr "%ssl_error_descr: %ssl_ca_name"
```

Comment:
```suggestion
msgid "%ssl_error_descr: %ssl_lib_error"
msgstr ""
```

Reply:
Should I assume that msgstr "" will get the values from the macros on the line above?

Reply:
Yes. If msgstr is empty the value in msgid will be displayed in the translation output.